.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature Request
+about: For Requesting enhancement or changes to the APIs functionality
+
+---
+
+## APIs Affected
+
+* [ ] VA Facilities
+* [ ] Benefits Intake
+* [ ] Appeals Status
+* [ ] Health
+* [ ] Veteran Verification
+
+## Brief Summary of the Request
+
+<!-- One or two sentence summary of the Request. -->
+
+## Acceptance Criteria
+
+<!-- List any criteria that developers can use to validate they've solved your request. -->
+
+* [ ] It should solve my problem
+* [ ] It should do it in a way that makes sense
```

Comment:
If these are examples, should they belong in the comment above?

Reply:
The text should probably be comments, but was hoping to help people who haven't used Markdown before figure out how to format.